### ISSUE TITLE
redirection notification is unreadable and transparent

### ISSUE THREAD

**Issue by genodeftest** [RequestPolicyContinued/requestpolicy#658]
using requestpolicy 1.0beta9.3

Since Firefox updated to version 38 the requestpolicy redirection notification is unreadable and transparent in case there is another notification. See attached screenshot for details.

![requestpolicy-notification-bug](https://cloud.githubusercontent.com/assets/934401/7787927/4647d8fe-0225-11e5-8447-b5526e7456bd.png)
In this case you can see a requestpolicy notification laying over a notification by CertPatrol.

**Comment by myrdd** [comment]
Could you please find a way how to reproduce this bug in a fresh profile?

**Comment by genodeftest** [comment]
Yes, the white (unreadable) font I can reproduce by these steps:
1. create a fresh firefox profile
2. install requestpolicy 1.0beta9.3
3. configure requestpolicy to block all requests by default
4. go to duckduckgo.com, search for something
5. open any link in new tab (e.g. with middle mouse click)

You will see the white font on light gray background.

To reproduce the transparency issue follow these steps:
1. create a fresh firefox profile
2. install requestpolicy 1.0beta9.3
3. configure requestpolicy to block all requests by default
4. install "certificate patrol" addon
5. open any link in new tab (e.g. with middle mouse click) which follows these rules:
* It should be on a https website you never visited before (new SSL certificate)
* It should redirect to another website
(I'll provide such a URL if I stumble upon one. In the mean time this should also work with a website trying to redirect and open a popup window at the same time)

**Comment by nodiscc** [comment]
Confirmed by combining RPC 1.0beta9.3 with https://addons.mozilla.org/en/firefox/addon/certificate-patrol/
